src/lib.rs

```diff
@@ -12,12 +12,20 @@
     clippy::match_like_matches_macro, // MSRV is lower than what's needed for matches!
     clippy::wrong_self_convention, // TODO fix code and remove
 )]
+// TODO: Fix coverage of examples from our README.md - apparently the incantation below is not
```

Comment:
Is the incantation broken already? or the changes in this MR broke? If it's the latter should be kept working

Maybe of interest https://github.com/RCasatta/bitcoind/pull/115 but possibly without the `doc` feature

Reply:
The incantation is broken already, because it is hidden behind a non-existent `"bench"` feature (e.g. the incantation is ignored / is a no-op).  In particular, I tried to inject an error into `README.md` (e.g. `let _ = 123 + "abc";`) and it didn't cause a `cargo test` failure.

Thank you for pointing out RCasatta/bitcoind#115 - this spelling does work (i.e. `cargo test` starts catching errors in examples that appead in the `README.md`).  But... this also ends up covering `README.md` by `cargo doc`:

* I am unsure if changing the actual text of crate-level doc is desirable.  The current crate-level doc ("This crate provides a QrCode encoder and decoder") seems nice and sufficient.  
* Additionally `cargo doc` complains about some errors:
    * "unresolved link to `*`" (this is about `decoder [*]`)
    * "this URL is not a hyperlink" (it suggests writing `<https://example.com>` instead of `https://example.com`)

So, in the end, I've left the README.md testing/coverage in the old state... :-/

Reply:
I generally like to have crate level and readme to be the same, but yeah this is not this PR fault, thanks for looking into it.